### PR TITLE
Make cumuls_based true by default

### LINKED ISSUE #743: make `cumulus_based` true by default
Default to `true` will cover most of the cases outside of the `polkadot` repo.

### PR DESCRIPTION
**Pull request by wirednkod**
Fixes #743

### CODE REVIEW

**Comment by pepoviola** [comment]
@wirednkod did you know why this pr don't run the test??

File: javascript/packages/orchestrator/src/configGenerator.ts [hunk 1]
```diff
@@ -350,11 +351,17 @@ export async function generateNetworkSpec(
         computedWasmCommand += ` > {{CLIENT_REMOTE_DIR}}/${GENESIS_WASM_FILENAME}`;
       }
 
+      // IF is defined use that value
+      // else check if the command is one off undying/adder otherwise true
+      const isCumulusBased = (parachain.cumulus_based !== undefined) ? parachain.cumulus_based :
+        [DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(collators[0].command!) ? false :
+        true;
```
Comment (wirednkod) [inline javascript/packages/orchestrator/src/configGenerator.ts]: Simplier this way:
```ts
      const isCumulusBased = (parachain.cumulus_based !== undefined) ? parachain.cumulus_based :
        ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(collators[0].command!);
```

Comment (pepoviola) [reply javascript/packages/orchestrator/src/configGenerator.ts]: nice!! :)

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
We should add this change as docs and maybe release as 1.4.0?

**Comment by wirednkod** [comment]
> We should add this change as docs and maybe release as 1.4.0?

Will add that and push a commit. Thank you

**Pull request merged**